email-service/internal/mail/backends/gomail.go

```diff
@@ -2,21 +2,29 @@ package backends
 
 import (
 	"context"
+	"errors"
 	"fmt"
 	"strconv"
 
-	"github.com/Hukyl/genesis-kma-school-entry/mail/config"
+	"github.com/GenesisEducationKyiv/software-engineering-school-4-0-Hukyl/email-service/internal/mail/config"
 	"github.com/go-gomail/gomail"
 )
 
 type GomailMailer struct {
 	config config.Config
 }
 
-func (gm *GomailMailer) SendEmail(_ context.Context, email, subject, message string) error {
+func (gm *GomailMailer) SendEmail(
+	_ context.Context, emails []string,
+	subject, message string,
+) error {
 	mail := gomail.NewMessage()
 	mail.SetHeader("From", gm.config.FromEmail)
-	mail.SetHeader("To", email)
+	if len(emails) == 0 {
+		return errors.New("no email recipients")
+	}
+	mail.SetHeader("To", emails[0])
+	mail.SetHeader("Bcc", emails[1:]...)
```

Comment:
are you sure, that you have more than one email in the slice? Below you made check that slice not empty, but there's still no guarantee that it has more than 1 receivers at this point

Reply:
what i mean - if there's only one element in `emails` then this code will panic

Reply:
1. I have unit tests that try with one email and they don't return an error.
2. Seems to be alright in [similar example](https://go.dev/play/p/VXFcoBe-XLP).

Reply:
ok, strange, my bad